Admin/AbstractAdmin.php

```diff
@@ -2845,6 +2852,22 @@ public function getDashboardActions()
     /**
      * {@inheritdoc}
      */
+    public function getSearchResultLink($object)
```

Comment:
> I can imagine a concreate case to override this method. Maybe the dev want's to check some other object properties to show a different route.

In this case, `$searchResultActions` could be useless IMHO.

Maybe this should be done on another method? Dunno. I would like @sonata-project/contributors though on this.